get_gcs_test.go

```diff
@@ -233,3 +233,35 @@ func TestGCSGetter_GetFile_OAuthAccessToken(t *testing.T) {
 	}
 	assertContents(t, dst, "# Main\n")
 }
+
+func Test_GCSGetter_ParseUrl_Malformed(t *testing.T) {
```

Comment:
It would be great to have a positive example for this test.

Reply:
Added a valid test here: https://github.com/hashicorp/go-getter/pull/512/files#diff-dde9a08c025c5bbf3c4aa8652fd9aa96700fbf222d4d315e6d71bcbc421c655dR237